GenderPayGap.WebUI/Views/AdminOrganisationSicCodes/ConfirmSicCodeChanges.cshtml

```diff
@@ -92,45 +89,59 @@
         else
         {
             <table class="govuk-table">
+                <thead class="govuk-table__head govuk-visually-hidden">
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header govuk-!-padding-0">
+                            <span class=" govuk-visually-hidden">SIC codes to add</span>

```

Comment:
You might not need this span if you're using `govuk-visually-hidden` on the `thead` (just put the text directly inside the `th`)?

Reply:
I think it looks nicer with the line above the first row and if I add class visually hidden to <th> then that line would be invisible.

Reply:
Ah ok, that's fair enough 😄